flink-dist/pom.xml

```diff
@@ -129,6 +129,12 @@ under the License.
 		<!-- Default file system support. The Hadoop and MapR dependencies -->
 		<!--       are optional, so not being added to the dist jar        -->
 
+		<dependency>
```

Comment:
with this you add the `flink-connector-files` to the `flink-dist` jar, no? So it is in the release two times?

Reply:
What makes you say is it in the release two times? My understanding is that in master flink-connector-files is included as part of the table uber jar, which is imported here and copied in `/lib`. With this PR instead, the flink-connector-files is not part of the table uber jar but is imported directly and copied to `/lib`.

I might be missing something here...

Reply:
Also in the tree here i see it only once: https://github.com/apache/flink/pull/17897#discussion_r760175589